lib/engines/content_block_manager/app/assets/javascripts/content_block_manager/modules/copy-embed-code.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+;(function (Modules) {
+  function CopyEmbedCode(module) {
+    this.module = module
```

Comment:
clarifying question: I've not used JS modules like this before... so does this `function` get called before the `init` below? and the `module` that's passed in is the html of the component?

Reply:
Pretty much, yeah, there's some docs here https://docs.publishing.service.gov.uk/repos/whitehall/javascript.html